templates/contact.twig

```diff
@@ -4,7 +4,6 @@
 
     <div class="container">
         <div class="post-content">
-            <div class="row">
```

Comment:
This did not have a closing tag, making the final markup of the page invalid. It caused the browser to include the footer within the post content div, which made it not full width anymore.